x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/predicate/operator/comparison/Equals.java

```diff
@@ -19,6 +20,12 @@
 import java.util.Map;
 
 public class Equals extends EsqlBinaryComparison implements Negatable<EsqlBinaryComparison> {
+    public static final NamedWriteableRegistry.Entry ENTRY = new NamedWriteableRegistry.Entry(
+        Expression.class,
+        "Equals",
```

Comment:
Nit: would feel refactoring-safer if we extracted the name from the class object, but I think we started like this and is quite unlikely the name's going to be changed, so it can stay as is. :)